lib/cache/ObservableCollection.js

```diff
@@ -209,7 +209,7 @@ export default class ObservableCollection {
      * @param docId
      */
     addById(docId) {
-        const doc = this.collection.findOne({ _id: docId }, this.options);
+        const doc = this.collection.findOne({ _id: docId }, { ...this.options, limit: 0, skip: 0 });
```

Comment:
Are you sure it's limit 0 and skip: 0? To avoid any weird issues in the future, I would simply strip out limit and skip for options
const { limit, skip, ...cleanedOptions } = this.options;

Reply:
On `skip` I think it makes sense, for `limit` it's defined as disabling it: https://docs.mongodb.com/manual/reference/method/cursor.limit/#zero-value

... but sure, better is to leave it out.